app/models/item.rb

```diff
@@ -25,4 +25,12 @@ def no_orders?
     item_orders.empty?
   end
 
+  def self.items_by_popularity(number, order = 'desc')
+    select("items.name, sum(item_orders.quantity) AS total_quantity")
+    .joins(:item_orders)
+    .where(active?: true)
+    .group(:name)
+    .order("sum(item_orders.quantity) #{order}")
+    .limit(number)
+  end
```

Comment:
Nice active record. I like how you changed the method to just list off items by popularity so that it can be used for the least popular too.